components/datadog/agent/helm.go

```diff
@@ -162,8 +193,12 @@ func NewHelmInstallation(e config.CommonEnvironment, args HelmInstallationArgs,
 	return helmComponent, nil
 }
 
-func buildLinuxHelmValues(installName, agentImagePath, agentImageTag, clusterAgentImagePath, clusterAgentImageTag string, clusterAgentToken pulumi.StringInput) pulumi.Map {
-	return pulumi.Map{
+type HelmValues struct {
```

Comment:
Any reason to embed? Why note `type HelmValues pulumi.Map`?

Reply:
You’re right : there’s no reason.
I’ve switched to type alias instead.